agent/engine/engine_integ_test.go

```diff
@@ -89,6 +89,7 @@ func setup(cfg *config.Config, t *testing.T) (TaskEngine, func(), credentials.Ma
 	}, credentialsManager
 }
 
+/*
```

Comment:
Why was this commented out?

Reply:
This part should be removed when we merge the enis branch, as it is moved to `test_common.go`,
 see [here](https://github.com/aws/amazon-ecs-agent/blob/dev/agent/engine/test_common.go#L26), but l forgot to remove this part, I will remove this part later.